desktop/src/apps/ProjectsApp/files/DocViewer.module.css

```
@@ -0,0 +1,307 @@
+  font-size: 1.05em;
+}
+
+.markdown p {
+  margin: 0.8em 0;
+}
+
+.markdown a {
+  color: var(--accent, #6ea8fe);
+  text-decoration: underline;
+  text-underline-offset: 2px;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)